ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -247,8 +247,9 @@ public Response handleKsqlStatements(
           request,
           distributedCmdResponseTimeout);
 
-      final KsqlRequestConfig requestConfig =
-          new KsqlRequestConfig(request.getRequestProperties());
+      final boolean internalRequest =
```

Comment:
Changing this to reduce the amount of logging from creating the KsqlRequestConfig

Reply:
Thanks! I didn't realize this was an option